frontend/src/theme/primitives/typography.ts

```diff
@@ -0,0 +1,39 @@
+export const fonts = {
+  sansSerif: "IBM Plex Sans, sans-serif",
+  mono: "IBM Plex Mono, monospace",
+}
+
+export const fontSizes = {
+  xxs: "0.375rem",
+  xs: "0.5rem",
+  sm: "0.75rem",
+  smDefault: "0.8rem",
+  md: "1rem",
+  lg: "1.25rem",
+  xl: "1.5rem",
+  xxl: "1.75rem",
+  xxxl: "2.25rem",
+  xxxxl: "3.125rem",
+}
+
+export const fontWeights = {
+  normal: 400,
+  medium: 500,
+  bold: 700,
+}
+
+export const lineHeights = {
+  normal: "normal",
+  none: "1",
+  tight: "1.25",
+  base: "1.6",
```

Comment:
probably something you might do as part of tables/dataframes but i see line heights of 1.3 and .75

Reply:
Cool. Will add there.

Reply:
Going to keep this, and adjust as we see new line heights.